### ISSUE TITLE
Executing File Exist Checker Not support glob

### ISSUE THREAD

**Issue by jcook-uptycs** [mszostok/codeowners-validator#195]
**Description**

These globs work in the CODEOWNERS file.
```
Jenkinsfile* @Team
Dockerfile* @Team
Makefile* @Team
```

This reports as 
> "make*.sh" does not match any files in repository
```
make*.sh @Team
```
This file does exist `scripts/utilities/makeAll.sh` in the repo.

**Expected result**

**Actual result**

**Steps to reproduce**

**Troubleshooting**
